### PR TITLE
feat: [STO-5962]: Use /harness folder inside containers

### PR DESCRIPTION
**Pull request by r0610205**

### CODE REVIEW

**Review by williamwissemann — APPROVED** [review]

File: handler/step.go [hunk 1]
```diff
@@ -65,10 +65,12 @@ func HandleStartStep(config *config.Config) http.HandlerFunc {
 			if v.Name == "harness" {
 				v.Name = hv.HostPath.Name
 				v.Path = hv.HostPath.Path
+				v.ContainerPath = hv.HostPath.ContainerPath
```
Comment (rutvijmehta-harness) [inline handler/step.go:68]: You might also need to set `s.StartStepRequestConfig.WorkingDir` if `ContainerPath` is not empty

Comment (r0610205) [reply handler/step.go:68]: That actually messes up Run step, since it's running on the host and relies on WorkingDir.

Comment (rutvijmehta-harness) [reply handler/step.go:68]: `ContainerPath` should only be populated to when step is running in the container

**Comment by r0610205** [comment]
Closed in favor of another approach

**Pull request closed without merging**